target_chains/ethereum/contracts/contracts/pulse/IPulse.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "./PulseEvents.sol";
+import "./PulseState.sol";
+
+interface IPulseConsumer {
+    function pulseCallback(
+        uint64 sequenceNumber,
+        address provider,
```

Comment:
In Pulse providers are not important and users shouldn't care about them.

Reply:
will remove Pulse altogether